src/server/app/util/templates/inactivity-email.server.view.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+
+<head></head>
+
+<body>
+<p>Hello  {{name}}, </p>
+
+<br>
+
+<p>It seems you haven't logged into your {{appName}} account in {{date}} days. Why not check in and see what's new!</p>
+<p>Have a question or just want to know what's new? Take a look at our Message of the Day page: </p>
+<p>{{ url }}</p>
+
+<br>
+
+<p>Keep in mind that all accounts that have been inactive for a period of at least 90 days are removed from the system.</p>
```

Comment:
Let's say 'deactivated' instead of 'removed from the system' since that's more accurate